### PR TITLE
New version: ONLYOFFICE.DesktopEditors version 7.3.3.99

### PR DESCRIPTION
**Pull request by papacarlo**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99758)

### CODE REVIEW

**Comment by romandemidov** [comment]
this is just a test

**Comment by BrandonWanHuanSheng** [comment]
Oops! There is no version about 7.3.3.99 When I check the url, https://github.com/ONLYOFFICE/DesktopEditors/releases/tag/v7.3.3
I always confirmed URL 2 times before submitting.

File: manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- pptx
+- rtf
+- txt
+- xls
+- xlsx
+- xps
+ReleaseDate: 2023-03-17
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v7.3.3.99/DesktopEditors_x86_test.exe
```
Comment (BrandonWanHuanSheng) [inline manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.installer.yaml:29]: This link is supposed to be https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v7.3.3/DesktopEditors_x86_test.exe

File: manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Created with WinGet Releaser v2 using Komac v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ONLYOFFICE.DesktopEditors
+PackageVersion: 7.3.3.99
```
Comment (BrandonWanHuanSheng) [inline manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.installer.yaml:5]: The installer version is 7.3.3 not 7.3.3.99

File: manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Created with WinGet Releaser v2 using Komac v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ONLYOFFICE.DesktopEditors
+PackageVersion: 7.3.3.99
```
Comment (BrandonWanHuanSheng) [inline manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.locale.en-US.yaml:5]: Not 7.3.3.99 Suppose is 7.3.3

File: manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- document
+- editor
+- excel
+- office
+- powerpoint
+- presentation
+- spreadsheet
+- text-editor
+- word
+ReleaseNotesUrl: https://github.com/ONLYOFFICE/DesktopEditors/releases/tag/v7.3.3.99
```
Comment (BrandonWanHuanSheng) [inline manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.locale.en-US.yaml:32]: The actual url is https://github.com/ONLYOFFICE/DesktopEditors/releases/tag/v7.3.3

File: manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+- rtf
+- txt
+- xls
+- xlsx
+- xps
+ReleaseDate: 2023-03-17
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v7.3.3.99/DesktopEditors_x86_test.exe
+  InstallerSha256: F2CA1BB6C7E907D06DAFE4687E579FCE76B37E4E93B7605022DA52E6CCC26FD2
```
Comment (BrandonWanHuanSheng) [inline manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.installer.yaml:30]: Checksum failed, decrypting the new hashes A quick verifying the hashes 
New hashes x86 is 20C71CF8099E89F0F013B9C062FBB7BCD94A9A95EE9761ACBD7DAFB729896D5C

File: manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+- Architecture: x64
+  InstallerUrl: https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v7.3.3.99/DesktopEditors_x64_test.exe
+  InstallerSha256: F2CA1BB6C7E907D06DAFE4687E579FCE76B37E4E93B7605022DA52E6CCC26FD2
```
Comment (BrandonWanHuanSheng) [inline manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.installer.yaml:33]: New hashes x64 is 141BCFD86FD7802813AEC3BA7A76296A5B40FF5AA8C4AF99005A956A4F725D03

File: manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+- xls
+- xlsx
+- xps
+ReleaseDate: 2023-03-17
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v7.3.3.99/DesktopEditors_x86_test.exe
+  InstallerSha256: F2CA1BB6C7E907D06DAFE4687E579FCE76B37E4E93B7605022DA52E6CCC26FD2
+- Architecture: x64
+  InstallerUrl: https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v7.3.3.99/DesktopEditors_x64_test.exe
```
Comment (BrandonWanHuanSheng) [inline manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.installer.yaml:32]: Actual link is "https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v7.3.3/DesktopEditors_x86.exe"

**Review by BrandonWanHuanSheng — COMMENTED** [review]
It look like which winget is duplicating the setup file Existed File Directory is winget-pkgs-master\winget-pkgs-master\manifests\o\ONLYOFFICE\DesktopEditors\7.3.3.50
winget-pkgs-master\winget-pkgs-master\manifests\o\ONLYOFFICE\DesktopEditors\7.3.3" Why it is causing to upgrade to reinstall. Default Manifest Path is winget-pkgs-master\winget-pkgs-master\manifests\o\ONLYOFFICE\DesktopEditors\7.3.3

**Comment by papacarlo** [comment]
Closed cause the PR was fired by a test commit

**Pull request closed without merging**